### ISSUE TITLE
Reason strings for SafeMath

### ISSUE THREAD

**Issue by rstormsf** [OpenZeppelin/openzeppelin-solidity#1241]
Curious if we should  have reason strings, so we can  user truffle v5 assertions.

Example:
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/math/SafeMath.sol#L22

```
    require(c / _a == _b, "SafeMath mul failed");
```

sometest.js

```js
try 
{
  const tx = await pool.sendTransaction({ from: contributor, value: changeValue})
} catch (err){
  assert.equal(err.reason, "SafeMath mul failed");
}
```

**Comment by nventuro** [comment]
We're going to move forward soon with reason strings (probably for either 2.1 or 2.2), see #888 for more discussion.

Closing as duplicate.

**Comment by rstormsf** [comment]
love it!
https://etherscan.io/address/0xC0c001140319C5f114F8467295b1F22F86929Ad0#code
@nventuro

**Comment by rhlsthrm** [comment]
Not seeing the require reasons in the latest code here: https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/math/SafeMath.sol#L41. How can we access the version with the require messages?

**Comment by nventuro** [comment]
@rhlsthrm they're being introduced [this PR](https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1709), and will be released in v2.3, for which a release candidate will be out tomorrow, and the actual release likely next week.
